src/main/kotlin/stringcalculator/view/CustomExpressionCalculatorView.kt

```diff
@@ -0,0 +1,14 @@
+package stringcalculator.view
+
+import stringcalculator.dto.InputDto
+import stringcalculator.dto.OutPutDto
+
+class CustomExpressionCalculatorView {
```

Comment:
싱글턴인 `object`가 더 어울릴것 같네요!
```suggestion
object CustomExpressionCalculatorView {
```